engine/swarm/admin_guide.md

```diff
@@ -263,13 +263,25 @@ manager node of a single-node swarm. It discards swarm membership information
 that existed before the loss of the quorum but it retains data necessary to the
 Swarm such as services, tasks and the list of worker nodes.
 
-### Joining a previously failed node
-
-If a node becomes unavailable, it cannot communicate with the rest of the swarm
-and its workload is redistributed among the other nodes.
-If access to that node is restored, it will join the swarm automatically, but it
-will join with no workload because the containers it was assigned have been
-reassigned. The node will only receive new workloads when the swarm is rebalanced.
-To force the swarm to be rebalanced, you can
-[update](../reference/commandline/service_update/) or
-[scale](../reference/commandline/service_scale/) the service.
+### Re-balancing a service after joining a new or previously failed node
+
+When you add a new node to a swarm, or a node re-joins after it has been
+unavailable, the new node does not automatically get a workload if the service
+is already running at the desired scale. Notably, when a failed node recovers
+and re-joins a swarm, the workloads it was previously running have been
+reassigned to other nodes, and it does not automatically take them back.
```

Comment:
Might be good to include reasoning behind _why_ this is the case, as I've found this behavior confusterates people.

@aluzzardi can give you more, but we don't place workload on the new server to prevent healthy services from being interrupted and to avoid dog piling newly joined nodes.